15_0_Setting_Up_a_Bitcoin_Regtest.md

```diff
@@ -0,0 +1,18 @@
+# Chapter 15: Setting Up a Bitcoin Regtest
+
+While developing Bitcoin applications, you might want to use your applications isolated from any public Blockchain such as the Mainnet or the Testnet.
+You can create a Blockchain from scratch using the Regtest, with one main advantage over Testnet: you choose when to create new blocks, so you have complete control over the environment.
```

Comment:
Decapitalize `Blockchain` (-> `blockchain`). This applies to the remainder of this PR.